users/reverseproxy.rst

```diff
@@ -75,9 +75,10 @@ Caddy v2
 
 .. code-block:: none
 
-    example.com
-
-    reverse_proxy /syncthing 127.0.0.1:8384
+    handle /syncthing/* {
```

Comment:
There is `handle_path`: https://caddyserver.com/docs/caddyfile/directives/handle_path

Reply:
Ok, changed to handle_path.